### PR TITLE
various

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

File: grails-app/views/templates/links/_accessPointLinksModal.gsp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <%@ page import="de.laser.ui.Btn" %>
 
 <g:if test="${editmode}">
-    <a role="button" class="${Btn.SIMPLE} ${tmplCss}" data-ui="modal" href="#${tmplModalID}">
+    <a role="button" class="ui button ${tmplCss}" data-ui="modal" href="#${tmplModalID}">
```
Comment (croqueGrec09) [inline grails-app/views/templates/links/_accessPointLinksModal.gsp:4]: reversal intended?

**Review by croqueGrec09 — COMMENTED** [review]
one question

File: grails-app/views/templates/links/_accessPointLinksModal.gsp (hunk 1 above)
Comment (klodav) [reply grails-app/views/templates/links/_accessPointLinksModal.gsp:4]: yes - structure should be revised overall

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**